snapshot_test.go

```diff
@@ -0,0 +1 @@
+package firego
```

Comment:
The branch is still undergoing some work. I made the PR because I lost track of changes a bit.